File: Ruby/time.rb
def input
  gets.to_i
end

def output(n)
  first = true
  n.times do
    num = gets.to_i
    puts num % 100 < 60 && num / 100 < 24 ? "YES" : "NO"
    first = false
  end
end

n = input
output(n)

File: Java/time.java
import java.util.Scanner;

class main{
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        for (int i = 0; i < n; i++){
            int num = scanner.nextInt();
            System.out.printf(num % 100 < 60 && num / 100 < 24 ? "YES " : "NO "); 
        }
    }
}